solr/core/src/java/org/apache/solr/servlet/HttpSolrCall.java

```diff
@@ -971,7 +971,7 @@ protected void writeResponse(
 
       if (solrRsp.getException() != null) {
         NamedList<Object> info = new SimpleOrderedMap<>();
-        int code = ResponseUtils.getErrorInfo(solrRsp.getException(), info, log);
+        int code = ResponseUtils.getErrorInfo(solrRsp.getException(), info, log, core);
```

Comment:
@sonatype-lift ignore

HttpSolrCall.init() is called during initialization. It will not race with HttpSolrCall.writeResponse(...). There is no response to be written before or during initialization.